### ISSUE TITLE
Game Crash with Ars Nouveau

### ISSUE THREAD

**Issue by walksanatora** [petrolpark/Destroy#381]
**Describe the bug**

Game Crash

**To Reproduce**

Steps to reproduce the behavior. For example:

1. have destroy and Ars Nouveau
2. I honestly dont know (it seems to involve the sky render somehow?

**Expected Behavior**

no crash

**System Information**

You should only report issues on the latest version of Destroy. No others are supported. Include the following information:

 - Create version 0.5.1f
 - Ars Nouveau 4.9.0

**Additional context**
crash report: https://mclo.gs/3iHfJip
debug.log: https://mclo.gs/HorEJY0

**Minecraft Username**

crash was not found by me but instead by Twisted_Code ( @macks2008 )

**Comment by walksanatora** [comment]
Twisted has given more more context about the situation when the crash occured
> It seemed from the crash report that it was unable to log me in because it couldn't get the pollution level during rain, somehow. I wonder if that would happen again, for anyone else, if logging in while raining...
I had also just re-spawned right before I disconnected/crashed (initially I clicked the close button but then it crashed as it was closing so I'm not sure which to consider it)

**Comment by Twisted-Code** [comment]
Yes hello, I crashed repeatedly while trying to connect to a game server hosted by one of our friends. I would @ mention her, but since it's a rendering crash I'm not sure she'd be able to add any more context than we've provided. Plus she's probably at work right now.
Steps to reproduce, from my point of view (but not verified, like I don't want to get stuck in another crash loop so... sorry):
1. Die
2. while respawning, possibly relevant, I noticed the server lagging enough that I couldn't see, almost like I was in the ground
3. I closed the game during this lag
4. I launched the game again
5. when trying to rejoin the server, I crashed, on two consecutive attempts. The crash report attached by walksanatora was one of the two crash reports, and the other is similar enough I can't tell them apart (same stacktrace)

My relatively uneducated hypothesis is that you are trying to access an object not yet initialized by the server join event?
